src/kernel/include/shared/cos_types.h

```diff
@@ -32,6 +32,16 @@ typedef u64_t         tcap_uid_t;
 typedef u32_t         sched_tok_t;
 #define PRINT_CAP_TEMP (1 << 14)
 
+/* Parameters for specifying resource requests from booter */
+typedef enum {
```

Comment:
Namespace should not be `REQ_`.  Should match the function name.  Perhaps `BOOT_HYP_`.

This should not be in `cos_types.h` which is for types that need to be shared between kernel and user-level (with some stupid exceptions).  If the functions above are in the booter library, then this should be there too.

The type name should match the namespace, e.g. `boot_hyp_` and should end with `_t`.  These mismatches are a little messy.